solax/inverter.py

```diff
@@ -82,19 +82,15 @@ def sensor_map(cls) -> Dict[str, Tuple[int, Measurement]]:
         Warning, HA depends on this
         """
         sensors: Dict[str, Tuple[int, Measurement]] = {}
-        for name, mapping in cls.response_decoder().items():
+        for idx, (name, mapping) in enumerate(cls.response_decoder().items()):
```

Comment:
My understanding is that you're implementing these changes to address #112 . However we need to be careful not to change the names of existing sensors in home-assistant, as this would be a breaking change. Please revert this for now. We need to implement this separately

Reply:
I reverted this change. Additionally, I commented out the numerical `run mode` sensor to not return two sensors with the same id. We should consider to also comment out line 71 of `x3_hybrid_g4.py` for the same reason: https://github.com/squishykid/solax/blob/9bc8d3d9b273e6950eb87c18cf72c57ba2bbb51a/solax/inverters/x3_hybrid_g4.py#L71-L72
Shall I raise another PR for this?

Reply:
Yep, feel free to raise a separate PR to address #112. I think the easy one would just be to remove the non-text-mode "run mode" sensor from x3_hybrid_g4. But if you implement something which can keep both and doesn't change the names for existing sensors, that's great!

Reply:
I raised an PR #122 to address this issue.